server/runtime/README.adoc

```diff
@@ -76,7 +76,6 @@ The following is an example server configuration:
 <cache-container default-cache="default" statistics="false">
       <transport cluster="${infinispan.cluster.name}" stack="udp"/>
       <global-state/>
-      <distributed-cache name="default"/>
```

Comment:
this configuration example is broken in multiple places. but if you remove the "default" cache, I have the following error:
```
09:56:11,281 FATAL (main) [org.infinispan.SERVER] ISPN080028: Infinispan Server failed to start org.infinispan.commons.CacheConfigurationException: ISPN000432: Missing configuration for default cache 'default' declared on container
```

Reply:
ah, yeah. cache-container default-cache="default"

Reply:
let's just make this example community only. I'll add back the default cache config.